src/simulate.jl

```diff
@@ -44,29 +81,28 @@ end
 """
 Simulate n scenario according to a given NoiseLaw
 
-
 Parameters:
 - law::Vector{NoiseLaw}
     Vector of discrete independent random variables
-    
+
 - n::Int
     number of simulations computed
 
+
 Returns :
 - scenarios Array(Float64,n,T)
     an Array of scenario, scenarios[i,:] being the ith noise scenario
 """
-
-function simulate(law::Vector{NoiseLaw},n::Int64)
-    if n <= 0 
+function simulate(law::Vector{NoiseLaw}, n::Int64)
+    if n <= 0
         error("negative number of simulations")
     end
     Tf = length(law)
-    scenarios = Array{Float64}(n,Tf)
+    scenarios = Array{Vector{Float64}}(n,Tf)
```

Comment:
You should take a look to the version of this which is on newsvendor-fix. Because after you deal with Array{Float64,3} in forwardsimulation when here it is still an array of vector